.github/workflows/build.yml

```diff
@@ -59,71 +59,66 @@ jobs:
           tag: ${{ github.ref }}
           file_glob: true
 
-  build-linux-clangpdb-gcc5:
-    name: Linux CLANGPDB/GCC5
+  build-linux-clangpdb:
+    name: Linux CLANGPDB
     runs-on: ubuntu-22.04
     steps:
       - uses: actions/checkout@v3
 
-      - name: Install Dependencies
-        run: |
-          sudo apt-get update
-          sudo apt-get install nasm uuid-dev libssl-dev iasl
-          wget https://apt.llvm.org/llvm.sh
-          chmod +x llvm.sh
-          sudo ./llvm.sh 13
-          echo "/usr/lib/llvm-13/bin" >> $GITHUB_PATH
+      - name: ./build_duet.tool
+        run: docker compose run -e TOOLCHAINS=CLANGPDB build-duet
+      - name: ./build_oc.tool
+        run: docker compose run -e TOOLCHAINS=CLANGPDB build-oc
 
-      - name: CI Bootstrap
-        run: |
-          src=$(/usr/bin/curl -Lfs https://raw.githubusercontent.com/acidanthera/ocbuild/master/ci-bootstrap.sh) && eval "$src" || exit 1
+      - name: Upload to Artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: Linux CLANGPDB Artifacts
+          path: Binaries/*.zip
 
-      - run: ./build_duet.tool
-      - run: ./build_oc.tool
+  build-linux-gcc5:
+    name: Linux GCC5
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: ./build_duet.tool
+        run: docker compose run -e TOOLCHAINS=GCC5 build-duet
```

Comment:
Are these running in separate containers? If so, build-oc will not see build-duet results, and thus builds will be incomplete.

Reply:
They indeed run in separate containers, but they both mount the host folder, so it should work.